src/features/mint/components/MintStatuses.tsx

```diff
@@ -74,12 +85,12 @@ export const MintDepositToStatus: FunctionComponent<MintDepositToProps> = ({
     let key = 0;
     if (timeRemained > 0) {
       key = showNotification(
-        <AddressValidityMessage
+        <GatewayAddressValidityMessage
           milliseconds={timeRemained}
           destNetwork={getChainConfigByRentxName(tx.destChain).full}
         />,
         {
-          variant: "warning",
+          variant: timeRemained < 6 * 3600 * 1000 ? "error" : "warning",
```

Comment:
Some sort of time utils to make it clearer that this is 6 hours would be nice

Reply:
introduced